caper/cromwell_backend.py

```diff
@@ -197,17 +215,32 @@ class CromwellBackendBase(UserDict):
     """
 
     TEMPLATE = {'backend': {'providers': {}}}
-    TEMPLATE_BACKEND = {'config': {'default-runtime-attributes': {}}}
-
+    TEMPLATE_BACKEND = {'config': {'default-runtime-attributes': {}, 'filesystems': {}}}
+    DEFAULT_CALL_CACHING_DUP_STRAT = [
+        CALL_CACHING_DUP_STRAT_SOFTLINK,
+        CALL_CACHING_DUP_STRAT_HARDLINK,
+        CALL_CACHING_DUP_STRAT_COPY,
+    ]
```

Comment:
This should probably be a `tuple` so it's immutable:
```suggestion
    DEFAULT_CALL_CACHING_DUP_STRAT = (
        CALL_CACHING_DUP_STRAT_SOFTLINK,
        CALL_CACHING_DUP_STRAT_HARDLINK,
        CALL_CACHING_DUP_STRAT_COPY,
    )
```

Reply:
Fixed as you suggested.